packages/now-cli/test/helpers/prepare.js

```diff
@@ -519,6 +519,13 @@ CMD ["node", "index.js"]`,
       'vercel.json': getConfigFile(true),
       'now.json': getConfigFile(true),
     },
+    'unauthorized-vercel-config': {
+      // This project is under the testing-internal team
+      '.vercel/project.json': JSON.stringify({
+        orgId: 'team_JgimPl9u9uauL7E4MjMLt605',
+        projectId: 'QmRoBYhejkkmssotLZr8tWgewPdPcjYucYUNERFbhJrRNi',
+      }),
+    },
```

Comment:
Only added this test case, rest is lint.

Reply:
@pacocoursey Can you run prettier once more since we reverted this parens feature?

Reply:
Missed this sorry. Done now!